### PR TITLE
New package: Nvidia.FrameView version 1.5

### LINKED ISSUE #195522: [Package Request]: FrameView
### How can we help?

None

### Please read and ensure the following

- [x] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://images.nvidia.com/content/geforce/technologies/frameview/FrameView_1.5/FrameViewSetup.exe
Publisher: Nvidia
Package Name: FrameView
Description: FrameView is an application for measuring frame rates, frame times, power, and performance-per-watt on a wide range of graphics cards. 
Package Version: 1.5
Installer Url: https://images.nvidia.com/content/geforce/technologies/frameview/FrameView_1.5/FrameViewSetup.exe

### PR DESCRIPTION
**Pull request by lemonjacob**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue? Resolve #195522

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/195520)

### CODE REVIEW

**Comment by Dragon1573** [comment]
Hi @lemonjacob 👋🏼, glad to see you resolve the problem by yourself 👍🏼.

Could you please modify your very first comment as follows, in order to link this pull request to the issue? When the PR is merged, the issue will be automatically closed. This can prevent other users from confusing the issue state.

```diff
  Checklist for Pull Requests
  
  * [x]  Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- * [x]  Is there a linked Issue? [[Package Request]: FrameView #195522](https://github.com/microsoft/winget-pkgs/issues/195522)
+ * [x]  Is there a linked Issue? Resolve #195522
```

**Comment by lemonjacob** [comment]
Should have fixed it

File: manifests/n/Nvidia/FrameView/1.5/Nvidia.FrameView.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Nvidia.FrameView
+PackageVersion: "1.5"
+InstallerType: exe
+InstallerSwitches:
+  Silent: -s
+Installers:
```
Comment (stephengillie) [inline manifests/n/Nvidia/FrameView/1.5/Nvidia.FrameView.installer.yaml:9]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — COMMENTED** [review]
HI @lemonjacob,

This package has a dependency on `VCRedist`:
image

**Comment by stephengillie** [comment]
Verify with dependency:
image

**Comment by lemonjacob** [comment]
Should work now for some reason when I edited it on GitHub it would change the file when I committed changes. It worked when I used GitHub desktop instead of github.com

**Review by stephengillie — APPROVED** [review]

**Pull request merged**